boards/storm/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    println!("cargo:rerun-if-changed=layout.ld");
+    println!("cargo:rerun-if-changed=../kernel_layout.ld");
```

Comment:
this is subtle, but I guess not a big deal... cargo picks up on `layout.ld` on it's own, but not on `kernel_layout`, since it's not in the package. So if we didn't have the layer of indirection I don't think we would wouldn't need these `build.rs`s